admin/src/components/ContributionDetail/index.tsx

```diff
@@ -36,23 +36,27 @@ class ContributionDetail extends React.Component<Props, State> {
 
     const renderDeetItem = (label: string, val: React.ReactNode) => (
       <div className="ContributionDetail-deet">
-        <div className="ContributionDetail-deet-value">
-          {val}
-        </div>
-        <div className="ContributionDetail-deet-label">
-          {label}
-        </div>
+        <div className="ContributionDetail-deet-value">{val}</div>
+        <div className="ContributionDetail-deet-label">{label}</div>
       </div>
     );
 
     const renderSendRefund = () => {
-      if (c.staking || !c.refundAddress || c.refundTxId || !c.proposal.isFailed || !c.user) {
+      if (
+        c.staking ||
+        !c.refundAddress ||
```

Comment:
Should this be removed? `refundAddress` shouldn't ever exist on a contribution, correct?

Reply:
ProposalContribution has a hybrid property that pulls that in from user.settings for the admin version of the schema.

Reply:
Cool! That probably helped to reduce changes.